frontend/app/components/menu.tsx

```diff
@@ -3,6 +3,7 @@ import { useState } from 'react';
 
 import { useTranslation } from 'react-i18next';
 
+import { DropdownMenu, DropdownMenuContent, DropdownMenuItem, DropdownMenuTrigger } from './dropdown-menu';
 import { InlineLink } from './inline-link';
```

Comment:
Might as well use `~/...` to import these components, since that's the convention we've used elsewhere.